src/Http/CookieFactory.php

```diff
@@ -72,45 +71,34 @@ public function __construct(Config $config)
      * This method returns a cookie instance for use with the Set-Cookie HTTP header.
      * It will be pre-configured according to Flarum's base URL and protocol.
      *
-     * @param  string  $name
-     * @param  string  $value
-     * @param  int     $maxAge
-     * @return \Dflydev\FigCookies\SetCookie
+     * @param  string       $name
+     * @param  string|null  $value
+     * @param  int|null     $maxAge
+     * @return SetCookie
      */
-    public function make($name, $value = null, $maxAge = null)
+    public function make(string $name, ?string $value, ?int $maxAge): SetCookie
```

Comment:
Any particular reasoning for the change from null default to nullable?

Reply:
And in the libraries source, it doesn't seem like $value is nullable https://github.com/hansott/psr7-cookies/blob/master/src/SetCookie.php#L36

Reply:
Our code allowed it, I didn't want to introduce even more changes than necessary. The nullable type hint is cleaner imo.